client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -162,12 +153,13 @@ export const addComments = ( { dispatch }, { query: { siteId, status } }, { comm
 	const byPost = groupBy( comments, ( { post: { ID } } ) => ID );
 
 	forEach( byPost, ( postComments, postId ) =>
-		dispatch( {
-			type: COMMENTS_RECEIVE,
-			siteId,
-			postId: parseInt( postId, 10 ), // keyBy => object property names are strings
-			comments: postComments,
-		} )
+		dispatch(
+			receiveComments( {
+				siteId,
+				postId: parseInt( postId, 10 ), // keyBy => object property names are strings
```

Comment:
Can we just use `+postId` for brevity?

Reply:
`parseInt` and `+` have slightly different semantics. 

`parseInt( '500 Hello There', 10 )` --> 500
`+ '500 Hello There'` --> NaN

That said, I'm pretty sure its fine in this case